rapid7_insightvm/plugin.spec.yaml

```diff
@@ -3696,11 +3863,13 @@ actions:
         title: Vulnerability Key
         description: The key to identify a specific instance if the type is Instance
         type: string
+        example: "example_key"
```

Comment:
Make this more realistic.

Reply:
I'll be honest, I don't even know what this would be